201-representing-CNAB-in-OCI.md

```diff
@@ -0,0 +1,132 @@
+
+Runtimes and registries MAY choose to _relocate_ the images and invocation images referenced in the bundle to the registry where the bundle is pushed, but in its simplest form, a CNAB bundle MAY be represented in an OCI registry by the canonical `bundle.json` file descriptor, referenced in an OCI index (or manifest list).
+
+![](https://i.imgur.com/PfTcKOm.png)
+
+### `bundle.json` manifest
+
+The `bundle.json` MUST be serialized as canonical JSON and MUST be stored in the registry as a blob. This blob MUST be referenced by its digest in an OCI manifest. The manifest media type SHOULD be `application/vnd.cnab.bundle.config.v1+json` but MAY be a standard container image config type (`application/vnd.oci.image.config.v1+json`) if the target registry does not support the CNAB media type.
+
+### Example
+
+The following non-canonical `bundle.json` file will be used as an example:
+
+```json
+{
+  "schemaVersion": "v1.0.0",
+  "name": "helloworld",
+  "version": "0.1.1",
+  "description": "A short description of your bundle",
+  "keywords": ["helloworld", "cnab", "tutorial"],
+  "maintainers": [
+    {
+      "name": "Jane Doe",
+      "email": "jane.doe@example.com",
+      "url": "https://example.com"
+    }
+  ],
+  "invocationImages": [
+    {
+      "imageType": "docker",
+      "image": "cnab/helloworld:0.1.1",
+      "size": 942,
+      "contentDigest": "sha256:a59a4e74d9cc89e4e75dfb2cc7ea5c108e4236ba6231b53081a9e2506d1197b6",
+      "mediaType": "application/vnd.docker.distribution.manifest.v2+json"
+    }
+  ],
+  "images": null,
+  "parameters": null,
+  "credentials": null
+}
+```
+
+The `bundle.json` is serialized into canonical JSON and is pushed as a blob to the registry. This blob is then referenced by its digest (for example `sha256:e91b9dfcbbb3b88bac94726f276b89de46e4460b55f6e6d6f876e666b150ec5b`) as the object config.
+
+The bundle is encapsulated in an OCI manifest which is then referenced as part of an OCI index (or manifest list):
+
+```json
+{
+  "schemaVersion": 2,
+  "config": {
+    "mediaType": "application/vnd.cnab.bundle.config.v1+json",
+    "digest": "sha256:e91b9dfcbbb3b88bac94726f276b89de46e4460b55f6e6d6f876e666b150ec5b",
+    "size": 498
+  },
+  "layers": null
+}
+```
+
+The bundle file MUST be stored in the OCI registry, as it is the canonical representation of a CNAB bundle, and is the object that is used to compute the signature for a thin bundle.
+While implementations MAY choose to surface information from the bundle as top-level annotations in the OCI representations, they MUST store the unmodified bundle file as a blob.
+
+The bundle file and registry location MAY be used by implementations to generate relocation maps at runtime.
+
+### Invocation images
+
+> In practice most images are built as Docker v2 images so we should allow these
```

Comment:
It is unclear what is being quoted here and again in the next section. Is this leftover from the editing process and can be removed?